eslint-plugin-expensify/no-object-keys-includes.js

```diff
@@ -0,0 +1,87 @@
+import CONST from './CONST.js';
+
+const message = CONST.MESSAGE.NO_OBJECT_KEYS_INCLUDES;
+
+const meta = {
+    type: 'problem',
+    docs: {
+        description: 'Disallow using Object.keys().includes() for performance reasons',
+        category: 'Performance',
+        recommended: true,
+    },
+    fixable: 'code',
+    messages: {
+        noObjectKeysIncludes: message,
+    },
+    schema: [],
+};
+
+function create(context) {
+    const sourceCode = context.getSourceCode();
+
+    /**
+     * Check if a node is Object.keys() call
+     */
+    function isObjectKeysCall(node) {
+        return (
+            node.type === 'CallExpression' &&
+            node.callee.type === 'MemberExpression' &&
+            node.callee.object.type === 'Identifier' &&
+            node.callee.object.name === 'Object' &&
+            node.callee.property.type === 'Identifier' &&
+            node.callee.property.name === 'keys' &&
+            node.arguments.length === 1
+        );
+    }
+
+    /**
+     * Get the text representation of a node
+     */
+    function getNodeText(node) {
```

Comment:
This function looks kind of useless to me. Let's remove it and do inline calls to `sourceCode.getText(node)` instead.